File: backend/src/main/java/com/NBE3_4_2_Team4/domain/board/genFile/entity/QuestionGenFile.java
package com.NBE3_4_2_Team4.domain.board.genFile.entity;

import com.NBE3_4_2_Team4.domain.base.genFile.entity.GenFile;
import com.NBE3_4_2_Team4.domain.board.question.entity.Question;
import jakarta.persistence.Entity;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@NoArgsConstructor
public class QuestionGenFile extends GenFile<Question> {}

File: backend/src/main/java/com/NBE3_4_2_Team4/domain/board/genFile/entity/QuestionGenFile.kt
package com.NBE3_4_2_Team4.domain.board.genFile.entity

import com.NBE3_4_2_Team4.domain.base.genFile.entity.GenFile
import com.NBE3_4_2_Team4.domain.board.question.entity.Question
import jakarta.persistence.Entity

@Entity
class QuestionGenFile : GenFile<Question>()
